unlock-app/package.json

```diff
@@ -30,6 +30,8 @@
     "@walletconnect/web3-provider": "1.7.8",
     "@xstate/react": "3.0.1",
     "apollo-boost": "0.4.9",
+    "axios": "1.0.0",
```

Comment:
Nooooo! We made all the effort to move to fetch!

Reply:
the generated client uses axios for cross-platform so it's already there. I can't use fetch with it so fetch usage will be replaced.